### DISCUSSION TITLE
Bluetooth del minipc non più visto dopo aggiornamento

### DISCUSSION THREAD

**Discussion by MarcoZ76** [indomus/forum#2673] [GENERALE - Discussioni generiche]
Ciao, ho un minipc NiPoGi dove è installato HA os,  funzionava l'hardware bluetooth del mini pc e lo vedevo come integrazione ma dopo un aggiornamento di HA di Agosto (10.5 ?) non viene più rilevato. Cosa posso controllare? Grazie

**Comment by indomus** [comment]
beh tu fai delle prove, magari riesci ad abbinarli anche "a distanza"

**Reply by MarcoZ76** [reply]
![IMG_4172](https://github.com/indomus/forum/assets/36272251/25873676-b264-467f-97c2-74fad6a03984)

**Reply by MarcoZ76** [reply]
Non trova l'hardware...  secondo me dopo l'aggiornamento il driver non viene più riconosciuto da HA

**Reply by MarcoZ76** [reply]
Che prove posso fare?

**Comment by Paoloc66** [comment]
Anche io stesso problema.
Se spegni il pc, stacchi e riattacchi l'alimentazione poi lo rivedi, ma dura pochi giorni...
Da quello che ho letto è un problema di HA che devono risolvere....
Se trovi qualche soluzione fai sapere...
